src/Umbraco.Cms.Integrations.Automation.Zapier/Constants.cs

```diff
@@ -42,5 +42,12 @@ public static class EntityType
 
             public const int Form = 2;
         }
+
+        public static class MediaAliases
```

Comment:
If you need to bump the version anyway (see separate comment), you can likely reference these constants from the core CMS.